docs/_docs/sql-reference/ddl.adoc

```diff
@@ -245,6 +245,60 @@ Drop Person table if the one exists:
 DROP TABLE IF EXISTS "Person";
 ----
 
+== CREATE INDEX
+
+Creates a new index.
+
+[source,sql]
+----
+CREATE INDEX [IF NOT EXISTS] name ON tableName
+[USING { HASH | TREE }]
+(column_name [ASC | DESC]
```

Comment:
Specification should reflect the fact that index might consists of several columns